src/test/java/seedu/address/ui/UiTestUtil.java

```diff
@@ -0,0 +1,58 @@
+    public static final int CHART_DEBT_BAR_INDEX = 0;
+    public static final int CHART_CREDIT_BAR_INDEX = 1;
+    public static final String TIME_PATTERN = "HH:mm";
+    public static final String DATE_PATTERN = "EEEE, MMMM d";
+    public static final String SECOND_PATTERN = ":ss";
+    public static final String HOME_CARD_TIME_FX_CODE = "#time";
+    public static final String HOME_CARD_DATE_FX_CODE = "#date";
+    public static final String HOME_CARD_SECOND_FX_CODE = "#second";
+    public static final String HOME_CARD_CHART_FX_CODE = "#chart";
+    public static final String HOME_CARD_AVAILABLE_LIST_FX_CODE = "#availableList";
+    public static final String HOME_CARD_CONTACT_FX_CODE = "#contactCount";
+
+    public static final String DISPLAY_CARD_TAG_FX_CODE = "#tags";
+    public static final String DISPLAY_CARD_DAY_FX_CODE = "#daysAvailable";
+    public static final String DISPLAY_CARD_PHONE_FX_CODE = "#phone";
+    public static final String DISPLAY_CARD_ADDRESS_FX_CODE = "#address";
+    public static final String DISPLAY_CARD_EMAIL_FX_CODE = "#email";
+    public static final String DISPLAY_CARD_BIRTHDAY_FX_CODE = "#birthday";
+    public static final String DISPLAY_CARD_MONEY_FX_CODE = "#moneyOwed";
+    public static final String DISPLAY_CARD_REMARK_FX_CODE = "#remark";
+    public static final String DISPLAY_CARD_NAME_FX_CODE = "#name";
+
+    public static final String DISPLAY_CARD_TAG_ICON_FX_CODE = "#tagIcon";
+    public static final String DISPLAY_CARD_DAY_ICON_FX_CODE = "#dayIcon";
+    public static final String DISPLAY_CARD_PHONE_ICON_FX_CODE = "#phoneIcon";
+    public static final String DISPLAY_CARD_ADDRESS_ICON_FX_CODE = "#addressIcon";
+    public static final String DISPLAY_CARD_EMAIL_ICON_FX_CODE = "#emailIcon";
+    public static final String DISPLAY_CARD_BIRTHDAY_ICON_FX_CODE = "#birthdayIcon";
+    public static final String DISPLAY_CARD_MONEY_ICON_FX_CODE = "#moneyIcon";
+
+    public static final String MINI_PERSON_CARD_NAME_FX_CODE = "#name";
+    public static final String MINI_PERSON_CARD_TAG_FX_CODE = "#tags";
+    public static final double CHART_BAR_GAP = 0.0;
+    public static final double CHART_CATEGORY_GAP = 120.0;
+    public static final boolean CHART_LEGEND_VISIBLE = false;
+    public static final String TESTFX_ROBOT = "testfx.robot";
+    public static final String TESTFX_HEADLESS = "testfx.headless";
+    public static final String PRISM_ORDER = "prism.order";
+    public static final String JAVA_HEADLESS = "java.awt.headless";
+    public static final String STRING_TRUE = "true";
+    public static final String STRING_GLASS = "glass";
+    public static final String STRING_SW = "sw";
+
+    public static void setUpSystem() {
+        System.setProperty(TESTFX_ROBOT, STRING_GLASS);
+        System.setProperty(TESTFX_HEADLESS, STRING_TRUE);
+        System.setProperty(PRISM_ORDER, STRING_SW);
+        System.setProperty(JAVA_HEADLESS, STRING_TRUE);
+    }
+}
```

Comment:
Is there suppose to be an empty line at the bottom for checkstyle?

Reply:
Yes, checkstyle was run for all modified files, thanks for clarifying Alvin!